cdap-ui/app/cdap/components/FastAction/SetPreferenceAction/SetPreferenceModal.js

```diff
@@ -226,11 +252,15 @@ export default class SetPreferenceModal extends Component {
 
   renderSpecifyPreferences() {
     const actionLabel = T.translate('features.FastAction.setPreferencesActionLabel');
-    let entity = "Namespace " + this.params.namespace;
+    let entity = this.params.namespace;
+    let entityWithType = `Namespace "${entity}"`;
```

Comment:
I think this adds `"` only for namespace preference modal. Don't think this adds it for programs and app right?
<img width="894" alt="screen shot 2017-01-30 at 11 46 01 am" src="https://cloud.githubusercontent.com/assets/1452845/22438685/c903d3f4-e6e1-11e6-9153-24460c215ad6.png">